azkaban-web-server/src/web/js/azkaban/view/schedule-sla.js

```diff
@@ -189,6 +189,14 @@ azkaban.ChangeSlaView = Backbone.View.extend({
       var duration = rFlowRule.cells[2].firstChild.value;
       var email = rFlowRule.cells[3].firstChild.checked;
       var kill = rFlowRule.cells[4].firstChild.checked;
+      if (email === false && kill === false) {
```

Comment:
Please use:
```
if (!email && !kill) {
        alert("....");
        return false;
}
```
`return false;` to stop event propagation, otherwise the code keeps running after clicking "Ok" in the alert  message and the API call is made anyways.

Reply:
In my testing, the API call was not made, but I agree. I will test this out. Thanks.